lib/msf/core/web_services/views/auth/login.erb

```diff
@@ -7,20 +7,56 @@
 </head>
 <body>
 
+<script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+<script>
+    function ConvertFormToJSON(form){
+        var array = jQuery(form).serializeArray();
+        var json = {};
+
+        jQuery.each(array, function() {
+            json[this.name] = this.value || '';
+        });
+
+        return JSON.stringify(json);
+    }
+
+    $(document).ready(function(){
+        $("#submit").on('click', function(){
+            $.ajax({
+                url: '<%= AuthServlet.api_generate_token_path %>',
```

Comment:
This should be `AuthServlet.api_login_path`, otherwise, the act of the user logging into the system generates a new token.

**`msfdb reinit` output:**
```
MSF web service user API token: 23474c55a38a3db370745e0a7c3f64a7f3271100c203a7795a407d2a761aec2dc18ed2dd4af79017
```

**API account display after logging in:**
```
Current API Token: bfea307a9be82456a00a661c4e0cfd850d0edec796ff042538b7e34ec19620ceedcb02f579a3dee5
```

Reply:
Regarding the associated verification step:
> Navigate to `/api/v1/auth/login`. Enter your username and password and verify you are given a valid token back. Also test the `Generate Net Token` link in the upper left corner to ensure you get a new token each time.

Logging into the API account should not generate a new API token. When the user logs in they should be viewing the current API token. Clicking the `Generate New API Token` link would generate a new API token for the user.

Reply:
We discussed this in depth privately, but just adding notes here:

I attempted to make this change, but ran into issues because the POST `/login` endpoint only returns 30x redirects from the server, whether the login request is valid or not. Javascript AJAX requests don't handle this well because as far as it's concerned both responses mean the same thing. 

We need to make some bigger changes and discussions around how we want to handle the login functionality. One of the key requests by @mkienow-r7 is to ensure the server has knowledge of the origin page that sent the user to the `/login` page so we can redirect them there after the login is successful. Implementing these changes is outside of the scope of this PR and will be made in the future.

As a stop-gap, the `/auth/login` page will always redirect to `/auth/account` on both valid and invalid login attempts. I did add some text to the `/auth/account` page to make it more apparent to the user that they are not currently logged in and redirect them back to the login page to try again.

Reply:
Thanks for making the note and created associated tickets @jbarnett-r7!